pyproject.toml

```diff
@@ -44,6 +44,7 @@ PyYAML = "^6.0"
 Jinja2 = "^3.0.3"
 black = "*"  # To format files in cli tools
 prometheus-client = "^0.16.0"
+requests-mock = "^1.10.0"
```

Comment:
Is already in `tool.poetry.group.dev.dependencies`, I don't think it should be installed when we need only the SDK